src/components/Tricots.js

```diff
@@ -3,50 +3,65 @@ import {Header, List, Image} from "semantic-ui-react";
 import {Helmet} from "react-helmet";
 import SingleTrikot from "./SingleTrikot";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'SingleTrikot'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)